test/RouteResultTest.php

```diff
@@ -24,32 +24,12 @@ public function setUp()
         };
     }
 
-    public function testRouteMiddlewareIsRetrievable()
-    {
-        $result = RouteResult::fromRouteMatch(
-            '/foo',
-            $this->middleware,
-            []
-        );
-        $this->assertSame($this->middleware, $result->getMatchedMiddleware());
-    }
-
     public function testRouteMiddlewareIsNotRetrievable()
     {
         $result = RouteResult::fromRouteFailure();
         $this->assertFalse($result->getMatchedMiddleware());
     }
 
-    public function testRouteRouteNameIsRetrievable()
```

Comment:
[Yes](https://github.com/michaelmoussa/zend-expressive-router/blob/1c7411f626919cffb901e22e34de76ead598e61a/test/RouteResultTest.php#L90-L102) (copy-pasta same link as other comment for completeness)